src/scripts/OSFramework/OSUI/Pattern/Rating/Rating.ts

```diff
@@ -142,6 +142,13 @@ namespace OSFramework.OSUI.Patterns.Rating {
 			// Check if the e.target is a label with the Enum.RatingCssClass.RatingInput class
 			const isInput = Helper.Dom.Styles.ContainsClass(currentTarget, Enum.CssClass.RatingInput);
 			if (isInput) {
+				const lastchosen = this.selfElement.querySelectorAll(GlobalEnum.HTMLElement.Input)[
```

Comment:
camelCase and _ prefix, as its private variable

Reply:
I know the rest of the code here doesn't follow that probably, so ignore this  if you want

Reply:
done